docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Troubleshooting.md

```diff
@@ -0,0 +1,77 @@
+# Troubleshooting Common Installation Errors
+
+This guide assumes installation on any supported OS.  
+
+This guide offers support for common issues faced when first installing LORIS. It is assumed that all installation steps provided have been 
+correctly followed and that all required dependencies have been installed.
+
+## Frontend Not Loading
+The first thing to try is to run:
+
+```bash
+make
+```
+
+Alternatively, on development instances:
+
+```bash
+make dev
+```
+
+If that does not solve your issue, this could be because the Config table in the database is not pointing to the correct values. In this case, follow the following steps. 
+
+1. Check the following values in the Config table via the MySQL backend.
+
+   | Name | Label          | Value          |  
+   | ---- | ----------- | ---------- |
+   | base | Base           | /var/www/loris |
+   | host | Host           | $yourHostName  | 
+    
+   These values can be viewed by running the following commands:
+    
+   ```sql
+    SELECT c.configID, cs.Name, cs.Label, c.value, cs.Description FROM Config c LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID) WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='base');
+    SELECT c.configID, cs.Name, cs.Label, c.value, cs.Description FROM Config c LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID) WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+   ```
+
+2. If they do not match the above format, update these values with the following commands on the MySQL command-line.
+
+   ```sql
+   UPDATE Config SET Value='$yourBasePath' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='base');
+   UPDATE Config SET Value='$yourHostName' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+   ```
+
+3. Check the base path does not have a trailing slash at the end. If it does, update its value to remove it.
+
+
+## Javascript changes not visible
+Try is to run:
+
+```bash
+make
+```
+
+Alternatively, on development instances:
+
+```bash
+make dev
+```
```

Comment:
It might be good to fold this section `Frontend Not Loading` since it's the same info. Maybe that header could read "Frontend Does Not Load/JavaScript changes not visible"

Reply:
Thanks! Addressed